lua/entities/gmod_wire_light.lua

```diff
@@ -227,7 +232,8 @@ function ENT:TriggerInput(iname, value)
 	self:UpdateLight()
 end
 
-function ENT:Setup(directional, radiant, glow, brightness, size, r, g, b, spritesize)
+function ENT:Setup(directional, radiant, glow, brightness, size, r, g, b, spritesize, on)
+	self.on = on or false
```

Comment:
You should be calling self:SetOn(on)

Reply:
`self.on` == whether the light can be turned on/off, `SetOn` == light is on.

Reply:
Lets get rid of the `self.on` then. All lights should be able to be turned on/off after this update I think. Right?

Reply:
I don't think so. The glow option is there for performance/visual configurability reasons. We don't need for users to configure whether the light can be turned on/off. We should just retain the default behavior from before (on being true by default).